### DISCUSSION TITLE
RTL sfp and IP changing issue

### DISCUSSION THREAD

**Discussion by Voidstranger** [Anime4000/RTL960x#170] [Q&A] [answered]
Recently Anime4000 suggested not changing default 192.168.1.1 lan ip address of RTL based sfp sticks, because of some "known issue":
"I suggest you do a factory reset, and try not change default IP Address, leave it as 192.168.1.1/24, this known issue most of the RTL based stick if not configured properly" 
Here: https://github.com/Anime4000/RTL960x/issues/169#issuecomment-1611769555

What is this issue? My router has plenty of "things" tied to default 192.168.1.1 router lan ip address. It may be a huge pain to reconfigure all of it.

**Comment by Anime4000** [comment, marked as answer]
RTL9601 D/CI firmware are very buggy, there are many hardcoded in the firmware,

you should start use non 192.168.0.0/22 IP pool space, maybe use 10.10.10.0/24 or something, that what network enthusiast do.

if you like 192.168.0.0/16, try use like 192.168.10.0, 192.168.99.0, aslong not default.

many high end consumer router like Asus start avoiding 192.168.1.1 as default
